src/Translator.java

```diff
@@ -125,17 +293,20 @@ private List<MIPSCommand> translateCallCommand(CallCommand command){
     }
 
     // TODO: float type operations
+    // TODO: float branching a need different MIPS command
     private List<MIPSCommand> translateBranchCommand(ConditionalBranchCommand command){
         List<MIPSCommand> commandList = new ArrayList<>(registerAllocator.enterCommand(command));
         Register a, b;
+
+        // FIXME: DUUUDE WTF IS THIS WHY NOT THE COMMENTED WAY BELOW BRO?
```

Comment:
cause i was doing some optimizations if both arguments were constants but ultimately scrapped that idea.